react-components/src/components/RevealTopbar/RuleBasedOutputsContainer.tsx

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useState, type ReactElement, useEffect, useMemo } from 'react';
+
+import { Button, Dropdown, Flex, Icon, Menu, Switch } from '@cognite/cogs.js';
+
+import { RuleBasedOutputs } from '../RuleBasedOutputs/RuleBasedOutputs';
+import { RULE_BASED_OUTPUTS_VIEW } from '../RuleBasedOutputs/constants';
+import { FdmSDK } from '../../utilities/FdmSDK';
+import { useSDK } from '../RevealCanvas/SDKProvider';
+import { type RuleAndEnabled } from '../RuleBasedOutputs/types';
+import { useTranslation } from '../i18n/I18n';
+
+export const RuleBasedOutputsContainer = (): ReactElement => {
+  const [ruleInstances, setRuleInstances] = useState<RuleAndEnabled[]>();
+  const [currentRuleSetEnabled, setCurrentRuleSetEnabled] = useState<any>();
```

Comment:
Can we have better typing for this?

Reply:
already changed and pushed here:
https://github.com/cognitedata/reveal/pull/4210/commits/e4c1637a2c6f4412f8fa79861d3d37844a646860